### DISCUSSION TITLE
trouble adding custom zeek script

### DISCUSSION THREAD

**Discussion by four80eastfan** [Security-Onion-Solutions/securityonion#1674] [Unsupported Versions]
I'm probably doing/missing something really silly, but when I run so-zeek-restart after modifying /opt/so/conf/zeek/local.zeek to load a custom script, local.zeek gets reset to default and my changes are gone.

EDIT: Security Onion 2.3.0

**Comment by cm-ops** [comment]
You will want to make your changes on the manager in the global or minion pillar file at /opt/so/saltstack/local/pillar/. There is a good description on how to do that here https://docs.securityonion.net/en/latest/zeek.html?highlight=local.zeek#configuration.
